schema/deploy/computed_columns/cif_user_full_name.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy cif:computed_columns/full_name to pg
+-- requires: tables/cif_user
+
+BEGIN;
+
+create function cif.cif_user_full_name(u cif.cif_user)
+returns varchar as $$
+  select u.family_name || ', ' || u.given_name;
```

Comment:
this will return `null` if either `family_name` or `given_name` is `null`. Check out `contact_full_name` to see how to handle `null` cases